docs/Overrides/NginxProxyManager.md

```diff
@@ -0,0 +1,98 @@
+# Nginx Proxy Manager
+[Nginx Proxy Manager](https://[https://nginxproxymanager.com/](https://nginxproxymanager.com/))  is  Docker container for managing Nginx proxy hosts and SSL Certificates with a simple, powerful interface.
+ 
+# Features of Nginx Proxy Manager
+ 
+### Get Connected
+Expose web services on your network · Free SSL with Let's Encrypt · Designed with security in mind · Perfect for home networks
+
+### Proxy Hosts
+Expose your private network Web services and get connected anywhere.
+
+### Beautiful UI
+Based on Tabler, the interface is a pleasure to use. Configuring a server has never been so fun.
+
+### Free SSL
+Built in Let’s Encrypt support allows you to secure your Web services at no cost to you. The certificates even renew themselves!
+
+### Multiple Users
+Configure other users to either view or manage their own hosts. Full access permissions are available.
+
+The GIT Repository for Nginx Proxy Manager is located at [[https://github.com/jc21/nginx-proxy-manager](https://github.com/jc21/nginx-proxy-manager)]
```

Comment:
Let's put the yaml for the override right after this. Right near the top after the title and description and links.

Reply:
Ok. This is corrected.